plugin/pkg/scheduler/schedulercache/cache_test.go

```diff
@@ -751,13 +816,21 @@ func benchmarkExpire(b *testing.B, podNum int) {
 	}
 }
 
-func makeBasePod(nodeName, objName, cpu, mem string, ports []v1.ContainerPort) *v1.Pod {
+func makeBasePod(nodeName, objName, cpu, mem, oir string, ports []v1.ContainerPort) *v1.Pod {
 	req := v1.ResourceList{}
 	if cpu != "" {
 		req = v1.ResourceList{
 			v1.ResourceCPU:    resource.MustParse(cpu),
 			v1.ResourceMemory: resource.MustParse(mem),
 		}
+		if oir != "" {
+			if len(strings.Split(oir, ":")) != 2 {
+				panic("Invalid OIR string")
```

Comment:
I think, we shouldn't panic here, rather return an error.

 https://github.com/golang/go/wiki/CodeReviewComments#dont-panic